README.md

```diff
@@ -26,49 +59,65 @@ const PersonSchema = z.object({
 	totalVisits: z.number(),
 });
 
-const person = generate(PersonSchema);
+const fixture = new Fixture();
+const person = fixture.from(PersonSchema);
 ```
 
-Gives you the following value for `person`:
+> You can also take a look at the [implemented example](./examples/create-fixture-person.test.ts).
```

Comment:
Could you also update the example to be an inline snapshot please.
The reason for this is that it's easier to read imho.